src/test/java/com/missafternoon/budjen/transactions/ConverterTest.java

```diff
@@ -0,0 +1,75 @@
+package com.missafternoon.budjen.transactions;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ConverterTest {
+
+    @Test
+    public void newTransactionCreated() {
+        String[] args = new String[]{"add", "--debit", "--description", "fish", "--amount", "500"};
+        Transaction transaction = new Converter().convert(args);
```

Comment:
Consider pulling the converter out to a private field

```java
private Converter target = new Converter();
```

and then changing this line to

```java
target.convert(args);
```

on this line and below